src/renderer/ui/utils/constants.js

```diff
@@ -42,9 +42,10 @@ export const ACTION_KEYS = _.transform(_.range(26), (final, current) => { // let
 })));
 
 export const languageMap = {
-  de: 'German',
-  'en-US': 'English',
+  cs: 'Czech',
   'nl-NL': 'Dutch',
```

Comment:
I fully understand this standard and have no problem adhering. Will revert the order. However it might be a minor usability issue if the list starts to grow. Perphaps a sorting function before the list is displayed would solve both problems cheaply.

Also are those standards published anywhere? I have only seen the participation guide for new pull requests.